src/cmdlinetest/test_help.t

```diff
@@ -62,3 +62,5 @@ Usage:
       -Q, --very-quiet    Very quiet mode; print only errors
       -c, --color         Force colored output
       -C, --no-color      Disable colored output
+      -f, --log-format    Define the format of timestamp in the log (default: no
```

Comment:
is it a log format or timestamp format?

Reply:
`--log-time-format` I think to stay consistent with Python logging which uses [formatTime](https://docs.python.org/3/library/logging.html#logging.Formatter.formatTime)

Reply:
Ok, adjusted it to `log_time_format`